src/lib/_loading.scss

```diff
@@ -104,45 +106,24 @@
     }
 }
 
-@keyframes bp-spinner {
-    0% {
-        transform: translate(-50%, -50%) rotate(0deg);
-    }
-
-    100% {
-        transform: translate(-50%, -50%) rotate(360deg);
+.bp .bp-doc.bp-doc-document {
+    .pdfViewer .page .loadingIcon {
```

Comment:
Is this level of specificity required? It also seems like the nesting is kind of arbitrary.

Reply:
the first level of nesting is our styles, the second level are the styles within PDFj.s' viewer that we are overriding. We could skip the first part but that if someone is using pdf.js directly elsewhere it would override their styles as well.